target/linux/realtek/files-6.12/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -131,7 +131,11 @@ static void rtl839x_create_tx_header(struct p_hdr *h, unsigned int dest_port, in
 static void rtl930x_create_tx_header(struct p_hdr *h, unsigned int dest_port, int prio)
 {
 	h->cpu_tag[0] = 0x8000;  /* CPU tag marker */
-	h->cpu_tag[1] = 0x0200; /* Set FWD_TYPE to LOGICAL (2) */
+
+	/* Set FWD_TYPE to LOGICAL (2)
+	 * ignore STP state for submission on specific port
+	 */
+	h->cpu_tag[1] = 0x0200 | BIT(2);
```

Comment:
Please use consistent assignment. BIT() | BIT() | BIT() preffered. Without constants make bits clear in comment. Same goes for 931x.